src/scrape_up/MediEncyclopedia/mediencyclopedia.py

```diff
@@ -0,0 +1,72 @@
+from html.parser import HTMLParser
+import bs4
+from bs4 import BeautifulSoup
+import requests
+
+"""
+        Create an instance of `MediEncyclopedia` class.
+
+    ```python
+    ency = MediEncyclopedia()
+    ```
+
+    | Methods               | Details                                                             |
+    | --------------------- | ------------------------------------------------------------------- |
+    | `.scrapebyurl()`      | Returns the medical dictation of associated topic url               |
+    | `.byletter()`         | Returns the list of medical relics starting with a particular letter|
+    
+
+    """
+
+
+class MediEncyclopedia:
+
+  def __init__(self):
+    print(self)
+
+  def scrapebyurl(self,url):
+    """Returns the medical data including references, review dates, content, topic-head and so on"""
+
+    try:
+      content = requests.get(url)
+      soup = BeautifulSoup(content.content, 'html.parser')
+      headline = soup.find('h1',attrs={'class':'with-also'}).text
+      article = soup.find('article')
+      for script in article(["script", "style"]):
+        script.extract()
+      text = article.get_text().replace('Browse the Encyclopedia','').replace('Clinical Trials','')
+      text = text.replace('To use the sharing features on this page, please enable JavaScript.','')
+
+      return [headline,text]
+    except:
+      return None  
+
+
+
+
+  def byletter(self, character):
```

Comment:
Provide a proper example in the docstring so that it is easier for the user to understand what kind of param they need to pass.

Also, have you tested this method?

Reply:
Okay, I'd change it accordingly. 
Also, I've added screenshots of testing of every method in screenshots section.

Reply:
Hi, I've added the proper docstrings, explaining each parameter's usage and need.